docs/docs-beta/docs/dagster-plus/deployment/environment-variables/dagster-ui.md

```diff
@@ -5,4 +5,115 @@ sidebar_position: 1
+
+</details>
+
+## How to find the environment variables page
+1. Sign in to your Dagster+ account
+2. Use the deployment switcher in the top right corner to select the deployment where you want to view or manage environment variables
+3. Navigate to the **Deployment** -> **Environment variables** page
+
+## How to add an environment variable
+To create a new environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **+ Add environment variable** button
+3. A modal will open, fill out the following information
+    - **Name** - Enter a name for the environment variable. This is how the variable will be referenced in your code.
+    - **Value** - Enter a value for the environment variable.
+    - **Deployment Scope** - select the deployment(s) where the variable should be accessible:
+        - **Full deployment** - The variable will be available to selected code locations in the full deployment
+        - **Branch deployments** - The variable will be available to selected code locations in Branch Deployments
+        - **Local** - If selected, the variable will be included when [exporting environment variables to a local `.env` file](#how-to-export-environment-variables-locally)
+    - **Code Location Scope** - select the code location(s) where the variable should be accessible. At least one code location is required.
+
+<!-- TODO SCREENSHOT -->
+
+4. Click **Save**
+
+## How to edit an environment variable
+To edit an existing environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **Edit** button on the right of the environment variable you want to edit
+3. A modal will open where you can edit the attributes of the environment variable
+4. Click **Save**
+
+## How to delete an environment variable
+To delete an environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **Trash icon** on the right of the environment variable you want to delete
+3. A modal will open to confirm that you want to delete the environment variable. Click **Yes, delete** to delete the environment variable, or **Cancel** to close the modal and not delete the environment variable.
+
+## How to view the value an environment variable
+To view the value of an environment variable:
+
+1. [Navigate to the environment variables page](#how-to-find-the-environment-variables-page)
+2. Click the **eye icon** in the **Value** column of the environment variable you want to view
+3. Click the **eye icon** again to hide the value.
+
+:::note
+Viewing an environment variable only reveals the value to you. It doesn't show the value in plaintext to all users. If you navigate away from the environment variables page or reload the page, the value will be hidden again.
+:::
+
+
+
+## How to add an environment variable for a Branch Deployment
```

Comment:
Does this need its own section?

Reply:
not really sure! before i wrote anything i started with a list of questions i thought people might have and one of them was "how do i set an env var for a branch deployment?" so i wrote this section. it is completely repetitive, but my thinking was like if you had that specific question you'd be able to go right to this section? happy to remove it though!

Reply:
read the section again, and i'm going to delete it